client/src/components/PasswordInput.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+const Input = styled.input`
```

Comment:
You can reuse https://github.com/lmachens/copypasta/blob/master/client/src/components/Input.js